finish/src/main/java/io/openliberty/guides/inventory/model/InventoryList.java

```diff
@@ -15,12 +15,15 @@
 import java.util.List;
 
```

Comment:
Remove this spacing to keep all imports in the same code block

Reply:
Updated.